bgcval2/timeseries/timeseriesTools.py

```diff
@@ -254,6 +257,69 @@ def getHorizontalSlice(nc, coords, details, layer, data=''):
     assert 0
 
 
+def save_csv(
+        timesD,
+        arrD, 
+        analysisname,
+        colours={},
+        linestyles={},
+        thicknesses={},
+        labels={},
+        name='',
+        units='',
+        region='',
+        layer='',
+        metric='',
+        title='',
+        ts='',
+        csvFolder='',
+        csvformat='.json',
+    ):
+    """
+    Output the data that appears in a plot as a csv.
+
+    """
+    if csvformat.lower() not in ['json', '.json']:
+        print('Format not set up', csvformat )
```

Comment:
```suggestion
        print('Looking for a json or .json file extension. Format not set up', csvformat )
```

Reply:
or 
```
raise TypeError
```
?

Reply:
yeah if you need it to exit right away, maybe OSError instead? TypeError refers to a type of a Python(-already) object that can not be operated on

Reply:
Changed